components/mpu60xx/examples/mpu_motion_detect/CmakeLists.txt

```diff
@@ -0,0 +1,9 @@
+# For more information about build system see
+# https://docs.espressif.com/projects/esp-idf/en/latest/api-guides/build-system.html
+# The following five lines of boilerplate have to be in your project's
+# CMakeLists in this exact order for cmake to work correctly
+cmake_minimum_required(VERSION 3.16)
+
+set(COMPONENTS main)
+include($ENV{IDF_PATH}/tools/cmake/project.cmake)
+project(mpu60xx_motion_detect)
```

Comment:
Thanks for the example!

Reply:
I believe any driver should have at least one example to demonstrate its use.
I'm waiting for some additional component, I'll be providing one more additional example once I receive it. 
And I'm sure it'll be liked even more by the team (I intend to add one more newly released driver from the repository in this one).

Reply:
We have some basic guidelines for examples in esp-idf project https://docs.espressif.com/projects/esp-idf/en/latest/esp32/contribute/creating-examples.html

The example overall looks good to me

Reply:
Thank you @tore-espressif. 
I'll follow this in next update.